org-ql.el

```diff
@@ -367,6 +367,7 @@ returns nil or non-nil."
                                             ;; It feels unintuitive that `find-file-noselect' returns
                                             ;; a buffer if the filename doesn't exist.
                                             (find-file-noselect it))
+                                          (get-buffer it)
```

Comment:
`org-ql-select` is not meant to be called interactively, and the docstring says that this value should be a list of buffers (not buffer names--admittedly, to Emacs, they are sometimes handled the same way).  So to support this, it should probably be done in `org-ql-search`'s `interactive` form.

Reply:
The current implementation expects each `it` in that typcase to be a file. So if I pass something like `("*scrathc*" "*scratch2*") it results in the `user-error: Can't open file: %s` on the next line. Which is why I added this. This is not directly related to the issue at hand though.

Reply:
The current implementation expects each `it` in that typcase to be a file. So if I pass something like `("*scrathc*" "*scratch2*")` it results in the `user-error: Can't open file: %s` on the next line. Which is why I added this. This is not directly related to the issue at hand though.